app/src/main/java/org/ole/planet/myplanet/ui/viewer/CSVViewerActivity.kt

```diff
@@ -21,16 +22,11 @@ class CSVViewerActivity : AppCompatActivity() {
     private fun renderCSVFile() {
         val csvFileOpenIntent = intent
         val fileName = csvFileOpenIntent.getStringExtra("TOUCHED_FILE")
-        if (!fileName.isNullOrEmpty()) {
```

Comment:
removing this line in all the activities will cause a crash incase the filename is empty